src/ptvsd/attach_server.py

```diff
@@ -23,6 +23,10 @@
 
 _pending_threads = set()
 
+_redirect_output_deprecation_msg = (
+    "'redirect_output' setting via enable_attach will be deprecated in the future versions of the debugger. "
+    "This can be set using redirectOutput in Launch config in VS Code, using Tee output option in Visual Studio, "
```

Comment:
Typo: `using Tee`

Reply:
The message says it will be deprecated in a future version, but I think it's already unused in this commit, no?

Reply:
The `using tee` part is talking about this setting in Visual Studio:
![image](https://user-images.githubusercontent.com/3840081/57317682-8ab0b380-70ad-11e9-991e-a6d48964a874.png)

About the deprecation text. Even though user could pass the redirection here, VS and VSC later also set it, always. so no matter what the user choice was here, it would always get overridden by the setting chosen in the IDE. Effectively having the same result as it being unused. Eventually we intend to remove this argument entirely.